packages/apps/human-app/frontend/src/modules/worker/jobs/my-jobs/components/desktop/my-jobs-job-type-filter.tsx

```diff
@@ -1,8 +1,8 @@
 /* eslint-disable camelcase --- ... */
 import { useTranslation } from 'react-i18next';
-import { useMyJobsFilterStore } from '@/modules/worker/hooks/use-my-jobs-filter-store';
 import { Filtering } from '@/shared/components/ui/table/table-header-menu.tsx/filtering';
 import { JOB_TYPES } from '@/shared/consts';
+import { useMyJobsFilterStore } from '../../../hooks';
 
 export function MyJobsJobTypeFilter() {
```

Comment:
Probably `filteringOptions` computation can be moved out of component (or `useMemo`) so `Filtering`  not getting re-rendered because of that.

Actually, the same thing happens when you declare anonymous function in a prop value. Ideally - we should handle that as well everywhere, but we can do it for all the code in separate PR